pkg/controller/manager/manager_controller.go

```diff
@@ -641,13 +648,6 @@ func (r *ReconcileManager) reconcileInstance(ctx context.Context, logc logr.Logg
 			TruthNamespace:  request.TruthNamespace(),
 			ServiceAccounts: []string{render.ManagerServiceAccount},
 			KeyPairOptions: []rcertificatemanagement.KeyPairOption{
-				// We need to render the certificate manager CA cert again in case we're running in multi-tenant mode.
-				// For single-tenant, this cert is created once by the core controller. For multi-tenant, we
-				// provision a unique CA per-tenant, and so we need to make sure to create it here.
-				//
-				// TODO: We probably want a separate tenant controller managing the creation of of this instead, so
```

Comment:
This was a hack I introduced in my last prototype PR that I am now replace with another hack in this PR (in the secrets_controller).

I think this probably belongs in a "Root CA controller" instead of either of these places.